src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/RazorBreakpointSpanResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable disable
+
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
+{
+    internal class RazorBreakpointSpanResponse
+    {
+        public RazorLanguageKind Kind { get; set; }
+
+        public Range Range{ get; set; }
```

Comment:
Given we don't need the extra code in the client we could probably remove the other properties leaving us with just "Range". Definitely still like having a holder object for Range in case we need to add stuff in the future